packages/trader/src/Modules/Trading/Containers/purchase.jsx

```diff
@@ -113,6 +124,7 @@ export default connect(
         validation_errors         : modules.trade.validation_errors,
         purchased_states_arr      : ui.purchase_states,
         setPurchaseState          : ui.setPurchaseState,
+        gtmStore                  : gtm,
```

Comment:
should be on line 116 after the client imports, since `g` is after `c` and before `m`

Reply:
sure done